tests/test_markup.py

```diff
@@ -93,21 +93,30 @@ def test_load_markup_error(query_factory, query):
 
 @pytest.mark.load
 @pytest.mark.system
-def test_load_system(query_factory):
+@pytest.mark.parametrize(
+    "query, entity_value, entity_type, entity_start_span, value, unit",
+    [
+        ("show me houses under {600,000 dollars|sys_amount-of-money}",
+         "600,000 dollars", "sys_amount-of-money", 21, 600000, "$"),
+        ("pay me {400|sys_amount-of-money}",
+         "400", "sys_amount-of-money", 7, 400, "unknown"),
```

Comment:
Let's add tests for one or more of these:
join the 2 p.m. meeting
Let's meet at 2 pm on Sunday, August 19 2018
 join the 11:39 meeting```

Reply:
done